module/VuFind/src/VuFind/Controller/AbstractRecord.php

```diff
@@ -252,6 +252,35 @@ public function deletetagAction()
      */
     public function homeAction()
     {
+        // If collections are active, we may need to check if the driver is actually
+        // a collection; if so, we should redirect to the collection controller.
+        $checkRoute = $this->params()->fromPost(
```

Comment:
Now that we have the `??` operator, I think it's actually more efficient to do something like `$checkRoute = $this->params()->fromPost('checkRoute') ?? $this->params()->fromQuery('checkRoute') ?? false;`. This way, we stop when we find a valid parameter, instead of having to check both parameters before deciding which one to use. (Nitpicky, I know).

Reply:
True, and more pretty. I didn't even realize how much nicer it is to chain parts with ??.